Cloud Application Manager/Dedicated Edition/camd-overview.md

```diff
@@ -0,0 +1,71 @@
+{{{
+"title": "Cloud Application Manager Dedicated Edition Overview",
+"date": "03-25-2020",
+"author": "Diego Sanjuan and Guillermo Sanchez",
+"keywords": ["cam", "cloud application manager", "Data Center edition", "appliance", "overview", "platforms", "dedicated", "cam-d"],
+"sticky": true,
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Supported Platforms](#supported-platforms)
+* [Installing Cloud Application Manager Dedicated Edition](#installing-cloud-application-manager-dedicated-edition)
+* [Next Steps](#next-steps)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview
+
+This article explains how to run Cloud Application Manager platform on-premise as a virtual appliance in your CenturyLink Private Cloud on VMware Cloud Foundation (CPC on VCF), VMWare data center or OpenStack cloud. After you install and [set up Cloud Application Manager Dedicated Edition](camd-initialsetup.md) for use, you’re ready to automate deployments to any cloud and platform.
+
+**Download the Cloud Application Manager Virtual Appliance**
+Request a download link from [support](mailto:incident@CenturyLink.com).
+
+Experience all the same functionality as the SaaS version, including managed services anywhere, except for some advanced features like cloud optimization. Use Cloud Application Manager Dedicated Edition to deploy either on private or public clouds.
+
+### Audience
+
+All Cloud Application Manager Dedicated Edition users who wants to deploy/install Cloud Application Manager Dedicated Edition (appliance).
+
+### Prerequisites
+
+* You should have a Data Center or AWS account to be able to deploy appliance in your preffered platform.
+
+### Supported Platforms
+
+Cloud Application Manager Dedicated edition is supported when deployed on these platforms:
+
+* CenturyLink Private Cloud on VMware Cloud Foundation (CPC on VCF)
+* VMware vSphere vCenter or Cloud Foundation data centers
+* OpenStack
+* AWS
+
+### Installing Cloud Application Manager Dedicated Edition
+
+For specific considerations on each platform, please refer to the corresponding link:
```

Comment:
Should it be `links` (plural)?

Reply:
Well, there is one link for each platform, that´s why it's singular.